Refresher/UI/PatchForm.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Refresher.UI;
 
-public abstract class PatchForm<TPatcher> : RefresherForm where TPatcher : Patcher
```

Comment:
Any particular reason for a `class` restraint?

Reply:
![image](https://github.com/LittleBigRefresh/Refresher/assets/34501060/030874e0-649a-4144-ac2d-c8d97355992b)
due to this error, i figured just doing a class restraint was better for readability over `this.Patcher = default(TPatcher)`, and we are unlikely to ever want a struct here anyway